godtools/App/Features/Lessons/Domain/Interface/GetLessonsListRepositoryInterface.swift

```diff
@@ -12,5 +12,4 @@ import Combine
 protocol GetLessonsListRepositoryInterface {
     
     func getLessonsListPublisher(appLanguage: AppLanguageDomainModel) -> AnyPublisher<[LessonListItemDomainModel], Never>
-    func observeLessonsChangedPublisher() -> AnyPublisher<Void, Never>
```

Comment:
Was able to remove the observer from the interface and move this to the class that implements the interface.